lib/geoengineer/resources/aws/iam/aws_iam_saml_provider.rb

```diff
@@ -6,7 +6,12 @@
 class GeoEngineer::Resources::AwsIamSamlProvider < GeoEngineer::Resource
   validate -> { validate_required_attributes([:name, :saml_metadata_document]) }
 
-  after :initialize, -> { _terraform_id -> { name } }
+  after :initialize, -> { _terraform_id -> { NullObject.maybe(remote_resource)._terraform_id } }
+  after :initialize, -> { _geo_id -> { name.to_s } }
```

Comment:
`_geo_id` in `_fetch_remote_resources` should be `name`

Reply:
Good catch, will change that!